src/callbacks/Login.ts

```diff
@@ -0,0 +1,16 @@
+import { saveToken } from "../secure-store/login/LoginToken";
+import { LoginParams, postLogin } from "../api/Auth";
```

Comment:
Consider making `api` and `secure-store` module as well, if u feel it will be imported frequently. 
For that, you have to register modules by editing  `tsconfig` and `babel.config`